docs/rules/await-async-utils.md

```diff
@@ -54,16 +52,13 @@ test('something correctly', async () => {
   );
 
   // `then` chained method is correct
-  waitForDomChange(() => {
-    return { container };
-  })
+  waitFor(() => {}, { timeout: 100 })
```

Comment:
Why adding a `timeout` to the example? 🤔

Reply:
It was just a stupid config object. I updated it on tests, I can do the same here.

Reply:
No, sorry. It's actually fine like this. This is a real example from new [`waitFor` optional second argument](https://testing-library.com/docs/dom-testing-library/api-async#waitfor).